docs/language-basics/imports.md

```diff
@@ -5,47 +5,47 @@ title: Imports
 
 # Import
 
-Imports add files or libraries to your Cadl program. When you compile a Cadl file, you provide a path to your root Cadl file, by convention called "main.cadl". From there, any files you import are added to your program. If you import a directory, Cadl will look for a `main.cadl` file inside that directory.
+Imports add files or libraries to your TypeSpec program. When you compile a TypeSpec file, you provide a path to your root TypeSpec file, by convention called "main.tsp". From there, any files you import are added to your program. If you import a directory, TypeSpec will look for a `main.tsp` file inside that directory.
 
-The path you import must either begin with `"./"` or `"../"` or otherwise be an absolute path. The path must either refer to a directory, or else have an extension of either ".cadl" or ".js". The following demonstrates how to use imports to assemble a Cadl program from multiple files:
+The path you import must either begin with `"./"` or `"../"` or otherwise be an absolute path. The path must either refer to a directory, or else have an extension of either ".tsp" or ".js". The following demonstrates how to use imports to assemble a TypeSpec program from multiple files:
 
-## Import Cadl file
+## Import TypeSpec file
 
-```cadl
-import "./models/foo.cadl";
+```typespec
+import "./models/foo.tsp";
 ```
 
 ## Import Js file
 
-```cadl
+```typespec
 import "./decorators.js";
 ```
 
 ## Import a library
 
-The import value can be name one of the package dependencies. In that case cadl will lookup for the `package.json` file and check the `cadlMain` entry (or default to `main` if absent) to decide what is the library entrypoint to load.
+The import value can be name one of the package dependencies. In that case typespec will lookup for the `package.json` file and check the `typespecMain` entry (or default to `main` if absent) to decide what is the library entrypoint to load.
 
-```cadl
-import "@cadl-lang/rest";
+```typespec
+import "@typespec/rest";
 ```
 
 ```json
-// ./node_modules/@cadl-lang/rest/package.json
+// ./node_modules/@typespec/rest/package.json
 {
-  "cadlMain": "./lib/main.cadl"
+  "typespecMain": "./lib/main.tsp"
```

Comment:
We might also consider tspMain for this one.

Reply:
I kind of like the expressiveness of typespecMain here. @bterlson , one swing vote here?

Reply:
The one thing I can say in favor of `tspMain` is that it matches the extension of the file we're referring to, though I don't have a strong opinion one way or the other.